### ISSUE TITLE
How to set NZB Password?

### ISSUE THREAD

**Issue by pquerner** [aforty/sabnzbd-knockstrap#21]
How does one set a password for a download? I cant figure it out.

**Comment by aforty** [comment]
Ruh-roh, one probably does not set a password for a download.

**Comment by pquerner** [comment]
Wow. x)
Are you considering of implementing this?

**Comment by aforty** [comment]
Yes, assuming it's a feature of SABnzbd's Plush theme (meaning that there's an API hook to do this). 

I just really haven't had the time to work on this project lately. There is a whole BS3 makeover that has been ongoing for months.

**Comment by pquerner** [comment]
Yep, the "plush theme" does have that feature (its a single-site for that nzb so to day, where you can set its category etc., see its parts and also set a archive password)

What do you mean by BS3?

**Comment by hashworks** [comment]
In the current master you can set passwords by adding them to the name of the download: `name{{password}}`
Will then become: `name/password`, meaning the password has been recognized.
